### PR TITLE
Brazil v1.0.0

### PR DESCRIPTION
**Pull request by robertoachar**
Translated HTML files and MD docs are placed in src/i18n/br.
I hope that it is useful.

### CODE REVIEW

**Comment by devongovett** [comment]
Sorry for just not knowing, but how different is Brazilian Portuguese from Portuguese from Portugal? Should we just set the language code to `pt` so that both get it, or just to `pt-br`?

**Comment by devongovett** [comment]
Also, thanks so much for doing this! 🎉

File: src/i18n/br/docs/code_splitting.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# ✂️ Separação do Código
+
+Parcel suporta separação do código sem nenhuma configuração, fora-da-caixa. Isso permite que você divida o código da sua aplicação em arquivos separados, que podem ser carregados sob demanda, o que significa tamanhos de arquivos iniciais menores e tempos de carregamento mais rápidos. À medida que o usuário navega em sua aplicação e os módulos são carregados, o Parcel automaticamente se encarrega de carregar os demais módulos sob demanda.
```
Comment (felipe-gdr) [inline src/i18n/br/docs/code_splitting.md]: I'd translate "out-of-the-box" differently here, as I think "fora-da-caixa" would look a bit weird to Brazilian readers. Maybe "de fábrica" sounds a bit more natural.

File: src/i18n/br/docs/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Parcel possui um servidor de desenvolvimento embutido, o qual irá reconstruir sua aplicação automaticamente à medida que você vá alterando os arquivos. Ele suporta [Substituição de Módulos à Quente](hmt.html) para acelerar o desenvolvimento. Apenas aponte à ele seu arquivo de entrada:
+
+```bash
+parcel index.html
+```
+
+Agora, abra seu navegador em http://localhost:1234/. Você também pode alterar a porta padrão utilizando a opção `-p <port number>`.
+
+Utilize o servidor de desenvolvimento caso você não tenha seu próprio sevidor, ou sua aplicação será inteiramente renderizada no cliente. Caso você possua seu próprio servidor, você pode utilizar o Parcel em modo `watch`. Essa opção continua reconstruindo sua aplicação à medida que você vá alterando os arquivos e suporta Substituição de Módulos à Quente, mas não irá iniciar um servidor.
```
Comment (felipe-gdr) [inline src/i18n/br/docs/getting_started.md]: "à medida que você **vai**"?

File: src/i18n/br/docs/hmr.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# 🔥 Substituição de Módulo à Quente
+
+A Substituição de Módulo à Quente, ou Hot Module Replacement (em inglês), ou simplesmente HMR, melhora a experiência de desenvolvimento atualizando automaticamente os módulos no navegador em tempo de execução sem precisar de uma atualização inteira da página. Isso significa que o estado do aplicativo pode ser mantido enquanto você muda pequenas coisas. A implementação do HMR do Parcel suporta arquivos de JavaScript e CSS fora-da-caixa. O HMR é automaticamente desativado quando se utiliza o Parcel em modo de produção.
```
Comment (felipe-gdr) [inline src/i18n/br/docs/hmr.md]: "de fábrica" > "fora-da-caixa"? (see comment above) :-)

File: src/i18n/br/docs/how_it_works.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# 🛠 Como Funciona
+
+O Parcel converte uma árvore de **recursos** em uma árvore de **pacotes**. Várias outras ferramentas de agrupamento de código são baseadas em arquivos de JavaScript, com outros formatos abordados - por exemplo, adicionadas como strings em arquivos JS. O Parcel é agnóstico à tipos de arquivo: ele funcionará com qualquer tipo de arquivos da maneira que você gostaria, sem configuração. Há três etapas para o processo de agrupamento no Parcel.
```
Comment (felipe-gdr) [inline src/i18n/br/docs/how_it_works.md]: "a tipos" doesn't take a grave accent.

File: src/i18n/br/docs/how_it_works.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+### 1. Construção da Árvore de Recursos
+
+Parcel aceita um único recurso como entrada, que pode ser qualquer tipo: um arquivo JS, HTML, CSS, imagem, etc. Existem vários [Tipos de Recursos](asset_types.html) definidos no Parcel que sabem como lidar com tipos de arquivos específicos. Os recursos são analisados, suas dependências são extraídas e são convertidos em sua forma compilada final. Isso cria uma árvore de recursos.
+
+### 2. Contrução da Árvore de Pacotes
```
Comment (felipe-gdr) [inline src/i18n/br/docs/how_it_works.md]: Typo: "Construção" instead of "Contrução"

File: src/i18n/br/docs/packagers.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+# 📦 Empacotadores
+
+No Parcel, um `Packager` combina múltiplos `Asset`s juntos em um pacote de saída final. Isso acontece no processo principal depois que todos os recursos foram processados ​​e a árvore de pacotes foi criada. Os `packagers` são registrados com base no tipo de arquivo de saída, e os recursos que geraram esse tipo de saída são enviados para esse pacote para a produção do arquivo de saída final.
```
Comment (felipe-gdr) [inline src/i18n/br/docs/packagers.md]: In this paragraph, you address Packager as simply "Packager", whereas on the previous paragraph you use a translated version "Empacotador". In order to maintain consistency, I'd use either "Packager" or "Empacotador" throughout the whole documentation.

**Comment by felipe-gdr** [comment]
@devongovett , BR and PT Portuguese, although very, very similar, have a few, important differences. I, being a Brazilian, think reading **pt_PT** texts is an awkward experience. I'm not sure if our fellow Portuguese friends have a similar opinion about reading **pt_BR** stuff. 

However, **pt_BR** is definitely better than **en** if you are from Portugal and doesn't have a fairly good level of English. So, maybe leaving it as **pt** is a good idea, at least until we get a **pt_PT** translation.

File: src/i18n/br/docs/code_splitting.md (hunk 1 above)
Comment (robertoachar) [reply src/i18n/br/docs/code_splitting.md]: This really makes it better. I will fix it.

File: src/i18n/br/docs/getting_started.md (hunk 2 above)
Comment (robertoachar) [reply src/i18n/br/docs/getting_started.md]: Thanks. I will fix it.

File: src/i18n/br/docs/hmr.md (hunk 3 above)
Comment (robertoachar) [reply src/i18n/br/docs/hmr.md]: This really makes it better. I will fix it.

File: src/i18n/br/docs/how_it_works.md (hunk 4 above)
Comment (robertoachar) [reply src/i18n/br/docs/how_it_works.md]: Thanks. I will fix it.

File: src/i18n/br/docs/how_it_works.md (hunk 5 above)
Comment (robertoachar) [reply src/i18n/br/docs/how_it_works.md]: Thanks. I will fix it.

File: src/i18n/br/docs/packagers.md (hunk 6 above)
Comment (robertoachar) [reply src/i18n/br/docs/packagers.md]: This is a specific case.

Packagers means "Empacotadores", or something like that. But if you look on the context, "Packager" is a class. That was my problem in translate it.

```js
class MyPackager extends Packager {...}
```

In this same documentation, we have both `Asset` and `Bundler` too.

Guess I need some help with this specific one.

**Comment by robertoachar** [comment]
@devongovett It was a great pleasure for me to translate this documentation. I take this opportunity to thank you for the excellent tool you have built.

@felipe-gdr Thank you immensely for the time you spent reviewing my work. I really appreciate that.

**Comment by devongovett** [comment]
@robertoachar @felipe-gdr Thanks so much for doing this, and sorry for the delay in getting it merged.

For now, I've renamed it to `pt` since it seems like that would be better than english if you are from Portugal. If someone wants to help point out the differences and make a translation for Portugal, that would be awesome. We'll need to do some work at that point to enable country-specific locales in the language detection code.

Check out https://pt.parceljs.org/!

**Comment by gnapse** [comment]
@devongovett perhaps this is not the best place to mention it, but I wanted to mention somewhere that I and a couple of colleagues are working on the Spanish version of the website. We plan to have it advanced enough to make a PR sometime next week (considering the holidays in between will hinder our ability to make progress faster). We're looking forward to have this done.

**Comment by robertoachar** [comment]
Hi @gnapse, such a great news. You're right, this is not the best place for it, but don't worry about that.

You could open an issue instead. If you have any questions, stay in touch.

File: src/i18n/br/index.html [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+    gtag('config', 'UA-110647385-1');
+  </script>
+</head>
+
+<body>
+  <header>
+    <img class="logo" src="assets/logo.svg" alt="Parcel">
+    <h2>Agregador de aplicações web absurdamente rápido, sem configuração</h2>
```
Comment (theuves) [inline src/i18n/br/index.html]: I think "agregador" and "absurdamente" aren't good translations.

Okay, that's a correct translation, but I think it should be something like:

> "Empacotador de aplicações web ultra-rápido, sem configuração"

(Very similar with the Spanish version.)

I don't speak things like "vou **agregar** minha aplicação.".

Comment (gnapse) [reply src/i18n/br/index.html]: Shoot out a PR maybe?

Comment (robertoachar) [reply src/i18n/br/index.html]: @theuves You're right. Both translations are better than mine. Will make a PR soon. Or you can do it. Your call. Thanks.

Comment (theuves) [reply src/i18n/br/index.html]: Okay, I can do it.

I'm making some changes in the translation and fixing some grammar errors.

Comment (theuves) [reply src/i18n/br/index.html]: #112

**Pull request merged**